src/components/NavBar/NavBar.js

```diff
@@ -4,8 +4,7 @@ import NavBarContainer from "./NavBarContainer";
 import { AvatarButton } from "./AvatarButton";
 import { UsernameLabel } from "./UsernameLabel";
 import { NavBarMenu } from "./NavBarMenu";
-import { compose, prop, defaultTo } from "lodash/fp";
-
+import { getUserName } from "../../utils/propertiesGetters";
```

Comment:
Rename from `propertieGetters` to `sessionInfo`

Reply:
For `sessionInfo`? Certainly I'm not satisfied using a Utils file approach

Reply:
Me neither, do you have any suggestion?